app/models/student.rb

```diff
@@ -33,9 +33,10 @@ def total_service_hours
 
   def self.to_csv
     CSV.generate do |csv|
-      csv << column_names
+      csv << column_names + (SEASONALS.map {|seasonal| "#{seasonal.to_s.pluralize}?"})
```

Comment:
Ruby style is usually `e.map { |x| x.something }`. Notice the spaces around the curly braces.